e2e-tests/LoginTest.js

```diff
@@ -6,32 +6,18 @@ fixture('Login')
 
 test('Test empty login', async t => {
   await t
-    .click(Selector('.btn-text').withText('Organisation'))
-    .expect(Selector('#login.modal').exists).ok(
-      'Login dialog failed to appear'
-    )
```

Comment:
Don't these changes effectively disable the test? It seems to me we should perhaps mock authentication during the test run to require certain dummy values.

Reply:
The login page no longer features a login modal, so I simply removed that part from the test.
As far as I can tell, we test as much now, as we did before.